sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -85,12 +85,12 @@ private void setAppExtras(SentryEvent event) {
   private SdkVersion getSdkVersion() {
     SdkVersion sdkVersion = new SdkVersion();
 
-    sdkVersion.setName("sentry-android");
+    sdkVersion.setName("sentry.java.android");
```

Comment:
so do we change here as well? now it's inconsistent, as it's supposed to be the maven packages if I got it right

Reply:
This is the SDK name, that'll be reported at the bottom of the event page.
Maven package goes on `packages`